src/Microsoft.IdentityModel.Protocols.WsFederation/GlobalSuppressions.cs

```diff
@@ -0,0 +1,34 @@
+[assembly: SuppressMessage("Design", "CA1055:Uri return values should not be strings", Justification = "Breaking change", Scope = "member", Target = "~M:Microsoft.IdentityModel.Protocols.WsFederation.WsFederationMessage.CreateSignOutUrl~System.String")]
+
+[assembly: SuppressMessage("Design", "CA1062:Validate arguments of public methods", Justification = "Checked in utility method", Scope = "member", Target = "~T:Microsoft.IdentityModel.Protocols.WsFederation.WsFederationMetadataSerializer")]
+
+[assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "Breaking change", Scope = "namespaceanddescendants", Target = "~T:Microsoft.IdentityModel.Xml.WsTrustConstants")]
+[assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "Breaking change", Scope = "type", Target = "~T:Microsoft.IdentityModel.Xml.WsTrustConstants_1_3")]
+[assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "Breaking change", Scope = "type", Target = "~T:Microsoft.IdentityModel.Xml.WsTrustConstants_1_4")]
+[assembly: SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "Breaking change", Scope = "type", Target = "~T:Microsoft.IdentityModel.Xml.WsTrustConstants_2005")]
+
+[assembly: SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Doesn't own object", Scope = "member", Target = "~M:Microsoft.IdentityModel.Protocols.WsFederation.WsFederationMetadataSerializer.WriteMetadata(System.Xml.XmlWriter,Microsoft.IdentityModel.Protocols.WsFederation.WsFederationConfiguration)")]
+[assembly: SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Doesn't own object", Scope = "member", Target = "~M:Microsoft.IdentityModel.Protocols.WsFederation.WsFederationMetadataSerializer.ReadMetadata(System.Xml.XmlReader)~Microsoft.IdentityModel.Protocols.WsFederation.WsFederationConfiguration")]
+[assembly: SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Doesn't own object", Scope = "member", Target = "~M:Microsoft.IdentityModel.Protocols.WsFederation.WsFederationMetadataSerializer.ReadEntityDescriptor(System.Xml.XmlReader)~Microsoft.IdentityModel.Protocols.WsFederation.WsFederationConfiguration")]
+
+[assembly: SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "Breaking change", Scope = "member", Target = "~P:Microsoft.IdentityModel.Protocols.WsFederation.SecurityTokenServiceTypeRoleDescriptor.KeyInfos")]
+[assembly: SuppressMessage("Security", "CA3075:Insecure DTD processing in XML", Justification = "Not created from factory, dtd is still prohibited", Scope = "member", Target = "~M:Microsoft.IdentityModel.Protocols.WsFederation.WsFederationMessage.GetTokenUsingXmlReader~System.String")]
```

Comment:
This is one we should consider writing a separate issue to investigate

Reply:
DTD behavior is prohibited as it should be, but this has other effects in parsing

Reply:
I opened: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/1501